addons/editor-number-arrow-keys/addon.json

```
@@ -1,5 +1,5 @@
 {
-  "name": "Arrow key incrementation in editor",
+  "name": "Editor arrow key incrementation",
```

The word "editor" in addon names is only necessary if there's a similar addon for another part of the website.
```suggestion
  "name": "Arrow key incrementation",
```